neural_compressor/tensorflow/algorithms/static_quant/keras.py

```diff
@@ -34,12 +34,12 @@
 
 def _add_supported_quantized_objects(custom_objects):
     """Map all the quantized objects."""
-    from .keras_utils.conv2d import QConv2D
-    from .keras_utils.dense import QDense
-    from .keras_utils.depthwise_conv2d import QDepthwiseConv2D
-    from .keras_utils.pool2d import QAvgPool2D, QMaxPool2D
-    from .keras_utils.quantizer import DeQuantize, FakeQuant, Quantize
-    from .keras_utils.separable_conv2d import QSeparableConv2D
+    from neural_compressor.tensorflow.keras.layers.conv2d import QConv2D
+    from neural_compressor.tensorflow.keras.layers.dense import QDense
+    from neural_compressor.tensorflow.keras.layers.depthwise_conv2d import QDepthwiseConv2D
+    from neural_compressor.tensorflow.keras.layers.pool2d import QAvgPool2D, QMaxPool2D
+    from neural_compressor.tensorflow.keras.layers.quantizer import DeQuantize, FakeQuant, Quantize
+    from neural_compressor.tensorflow.keras.layers.separable_conv2d import QSeparableConv2D
 
```

Comment:
Suggest exposing all layers under `neural_compressor.tensorflow.keras`, so that we can access them directly as below:

```python
from neural_compressor.tensorflow.keras.layers import QConv2D, QDepthwiseConv2D ...
```